impeller/entity/shaders/glyph_atlas_color.frag

```diff
@@ -16,5 +21,9 @@ out f16vec4 frag_color;
 
 void main() {
   f16vec4 value = texture(glyph_atlas_sampler, v_uv);
-  frag_color = value * v_text_color.aaaa;
+  if (frag_info.use_text_color == 1.0f) {
```

Comment:
Nit: The floating point comparison makes me nervous. Perhaps `frag_color = mix(value * v_text_color.aaaa, value.aaaa * v_text_color, frag_info.use_text_color)` instead?

Reply:
Why does it make you nervous?

Reply:
I'm going to merge this now. Please file a bug describing the problem you'd like me to address and I'll handle it later.